### PR TITLE
fix(speech): increment retry setting values and test timeout values in IT

### PR DESCRIPTION
**Pull request by diegomarquezp**
For resolving the speech IT failures produced in the integration tests of https://github.com/googleapis/google-cloud-java/pull/9171

Example failure log
https://fusion2.corp.google.com/ci/kokoro/prod:cloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fpresubmit%2Fintegration/activity/80fe9394-262e-47f6-94ad-37354783a7e8/summary

### CODE REVIEW

File: java-speech/README.md [hunk 1]
```diff
@@ -14,7 +14,6 @@ Java idiomatic client for [Cloud Speech][product-docs].
 
 If you are using Maven, add this to your pom.xml file:
 
-<!--- {x-version-update-start:google-cloud-speech:released} -->
```
Comment (meltsufin) [inline java-speech/README.md:17]: @ddixit14 Why is OwlBot removing this?

**Comment by burkedavison** [comment]
Is this in response to an issue with speech IT flakiness? The nightlies haven't been showing an issue here - but is it showing up elsewhere?

**Review by meltsufin — APPROVED** [review]
Please reference an issue in the description, if there is one for this flake.

**Comment by diegomarquezp** [comment]
@meltsufin @burkedavison I added context in the description

**Comment by meltsufin** [comment]
@diegomarquezp This seems similar to an issue that was reported by a customer in the past: https://github.com/googleapis/java-speech/issues/207.
Please make sure you're applying the [suggestion there](https://github.com/googleapis/java-speech/issues/207#issuecomment-661305530) that seems to have worked.

File: java-speech/google-cloud-speech/src/test/java/com/google/cloud/speech/v1/it/ITSpeechTest.java [hunk 2]
```diff
@@ -44,17 +46,22 @@ public class ITSpeechTest {
...
-                .setInitialRetryDelay(Duration.ofSeconds(5))
-                .setMaxAttempts(5)
-                .setMaxRetryDelay(Duration.ofSeconds(5))
+                .setInitialRetryDelay(Duration.ofMillis(500L))
+                .setRetryDelayMultiplier(1.5)
+                .setMaxRetryDelay(Duration.ofMillis(5000L))
+                .setInitialRpcTimeout(Duration.ZERO)
+                .setRpcTimeoutMultiplier(1.0)
+                .setMaxRpcTimeout(Duration.ZERO)
+                .setTotalTimeout(Duration.ofDays(1))
```
Comment (meltsufin) [inline java-speech/google-cloud-speech/src/test/java/com/google/cloud/speech/v1/it/ITSpeechTest.java:59]: This seems like a pretty long timeout.

Comment (lqiu96) [reply java-speech/google-cloud-speech/src/test/java/com/google/cloud/speech/v1/it/ITSpeechTest.java:59]: I believe the default timeout set to be 5 min. I think you can probably just double it to 10 minutes and *should* be fine. There was an issue from the java-doc-samples repo a few weeks ago about a similar LRO issue and the runtime recently increased for ~4:30min to ~5:30min.

Comment (diegomarquezp) [reply java-speech/google-cloud-speech/src/test/java/com/google/cloud/speech/v1/it/ITSpeechTest.java]: Using a more reasonable `ofHours(1)`

Comment (diegomarquezp) [reply java-speech/google-cloud-speech/src/test/java/com/google/cloud/speech/v1/it/ITSpeechTest.java]: I did not see your suggestion @lqiu96 . Let's try 10 minutes

**Review by meltsufin — APPROVED** [review]

**Pull request merged**